### DISCUSSION TITLE
Pipeline Outputs Result Endpoint ?

### DISCUSSION THREAD

**Discussion by amixyas** [kubeflow/pipelines#10670] [General]
Hi!  

Please when I add some outputs for the pipeline ( means : outputs comming for a task ( means : form a component ) ) I don't know which kfp.endpoint that I should call to get the results of this outputs .. the kubeflow web app display the outputs but I can't find which endpoint you're calling to get this results ? 

The pipeline_spec of then run does not contain any outputs except the type.

Thanks in advance for the help!

**Comment by amixyas** [comment]
![image](https://github.com/kubeflow/pipelines/assets/61936466/41b5b2ff-12cb-4156-a70e-8380ee940ff6)

**Comment by rimolive** [comment]
Are you using KFPv1 or KFPv2?

**Reply by amixyas** [reply]
KFPv2

**Reply by rimolive** [reply]
This [sample](https://github.com/kubeflow/pipelines/blob/master/samples/tutorials/Data%20passing%20in%20python%20components/Data%20passing%20in%20python%20components%20-%20Files.py) explains everything about data passing using Kubeflow Pipelines.
